runtime/kusama/src/weights/runtime_parachains_hrmp.rs

```diff
@@ -87,17 +87,17 @@ impl<T: frame_system::Config> runtime_parachains::hrmp::WeightInfo for WeightInf
 	/// The range of component `i` is `[0, 127]`.
 	/// The range of component `e` is `[0, 127]`.
 	fn force_clean_hrmp(i: u32, e: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 15_000
-			.saturating_add(Weight::from_ref_time(7_134_000 as u64).saturating_mul(i as u64))
-			// Standard Error: 15_000
-			.saturating_add(Weight::from_ref_time(7_238_000 as u64).saturating_mul(e as u64))
-			.saturating_add(T::DbWeight::get().reads(2 as u64))
-			.saturating_add(T::DbWeight::get().reads((2 as u64).saturating_mul(i as u64)))
-			.saturating_add(T::DbWeight::get().reads((2 as u64).saturating_mul(e as u64)))
-			.saturating_add(T::DbWeight::get().writes(4 as u64))
-			.saturating_add(T::DbWeight::get().writes((3 as u64).saturating_mul(i as u64)))
-			.saturating_add(T::DbWeight::get().writes((3 as u64).saturating_mul(e as u64)))
+		Weight::from_ref_time(860_276_000 as u64)
+			// Standard Error: 78_207
+			.saturating_add(Weight::from_ref_time(2_650_307 as u64).saturating_mul(i as u64))
+			// Standard Error: 78_207
+			.saturating_add(Weight::from_ref_time(2_689_181 as u64).saturating_mul(e as u64))
+			.saturating_add(T::DbWeight::get().reads(256 as u64))
+			.saturating_add(T::DbWeight::get().reads((1 as u64).saturating_mul(i as u64)))
+			.saturating_add(T::DbWeight::get().reads((1 as u64).saturating_mul(e as u64)))
+			.saturating_add(T::DbWeight::get().writes(384 as u64))
```

Comment:
We cannot use that. Same problem as here https://github.com/paritytech/cumulus/pull/1784  
And should be fixed in [1125011be165fca564fb78c8a50124537a1f4d0e](https://github.com/paritytech/substrate/pull/12482)